1-100.py

```diff
@@ -0,0 +1,20 @@
+import random
+
+
+def ask_and_check(rand_number):
+    answer = int(input())
+    if answer > rand_number:
+        print("Za duża liczba.")
+    elif answer < rand_number:
+        print("Za mała liczba.")
+    else:
+        print("Tak! Brawo!")
+        return 0
```

Comment:
nie ma tutaj potrzeby zwracania 0. To nie funkcja main w C/C++, że musimy tutaj zwrócić poprawny status zakończenia programu. Ta wartość jest tutaj zwracana w próźnie.